features/include/pcl/features/impl/rops_estimation.hpp

```diff
@@ -193,17 +193,19 @@ pcl::ROPSEstimation <PointInT, PointOutT>::computeFeature (PointCloudOut &output
       } while (theta < 90.0f);
     }
 
-    float norm = 0.0f;
-    for (std::size_t i_dim = 0; i_dim < feature_size; i_dim++)
-      norm += std::abs (feature[i_dim]);
+    const float norm = std::accumulate(
+        feature.cbegin(), feature.cend(), 0.f, [](const auto& sum, const auto& val) {
+          return sum + std::abs(val);
+        });
+    float invert_norm;
     if (norm < std::numeric_limits <float>::epsilon ())
-      norm = 1.0f;
+      invert_norm = 1.0f;
     else
-      norm = 1.0f / norm;
+      invert_norm = 1.0f / norm;
 
-    output.points.emplace_back ();
+    output.points.emplace_back (PointOutT());
```

Comment:
Can you explain this change?

Reply:
This is an error. Reverting it. I think I put in when compiler was spewing errors and it went ahead in the commit